apps/projects/app/components/Panel/PanelComponents.js

```diff
@@ -71,26 +71,24 @@ const statuses = [
   { color: 'positive', text: 'Accepted' },
 ]
 
-export const Status = ({ review }) => {
+export const Status = ({ reviewDate, approved }) => {
   const theme = useTheme()
-  const status = statuses[Number(review.approved)]
+  const status = statuses[Number(approved)]
   const text = status.text
   const color = theme[status.color]
   return (
     <div css="display: flex; align-items: center">
       <IconCheck color={color} css={`margin-top: -${0.5 * GU}px; margin-right: ${GU}px`} />
       <Text color={`${color}`}>{text}</Text>
       <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
-        {formatDate(new Date(review.reviewDate), 'd MMM yy, h:MM a (z)')}
+        {formatDate(new Date(reviewDate), 'd MMM yy, h:MM a (z)')}
       </Text>
     </div>
   )
 }
 Status.propTypes = {
-  review: PropTypes.shape({
-    approved: PropTypes.bool.isRequired,
-    reviewDate: PropTypes.string.isRequired,
-  }).isRequired,
+  reviewDate: PropTypes.string.isRequired,
+  approved: PropTypes.bool.isRequired,
```

Comment:
Why switch the order of these to be non-alphabetical?

Reply:
Fixed.